src/Event/Event.php

```diff
@@ -120,7 +120,7 @@ public function __set($attribute, $value)
     {
         $method = 'set' . ucfirst($attribute);
         deprecationWarning(
-            "Event::${$attribute} is deprecated. " .
+            "Event::\${$attribute} is deprecated. " .
```

Comment:
I would be more the friend of `sprintf('%s is deprecated.', 'Event::' . $attribute)` etc to clearly separate code and text parts. And also to avoid the escaping and  {} overhead then.